sources/core/Stride.Core.CompilerServices/Analyzers/STRDIAG005ReadonlyMemberIsReferenceType.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp;
+using System.Runtime.Serialization;
+using Stride.Core.CompilerServices.Common;
+
+namespace Stride.Core.CompilerServices.Analyzers;
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class STRDIAG005ReadonlyMemberIsReferenceType : DiagnosticAnalyzer
```

Comment:
Name is incorrect - suggestion `ReadonlyMemberTypeIsNotSupported`

Reply:
[e4fa7d4](https://github.com/stride3d/stride/pull/1864/commits/e4fa7d45fe90b9cb68cb2611a508111fbd517829)